xbmc/utils/StringUtils.h

```diff
@@ -54,9 +56,26 @@ class StringUtils
   \param ... variable number of value type arguments
   \return Formatted string
   */
-  static std::string Format(PRINTF_FORMAT_STRING const char *fmt, ...) PARAM1_PRINTF_FORMAT;
+  template<typename... Args>
+  static std::string Format(const std::string& fmt, const Args&... args)
```

Comment:
Yeah you're right. That's what you get from years of const-ref'ing everything :-/ This is exactly what universal references are for.